base_business_document_import/models/business_document_import.py

```diff
@@ -103,6 +104,19 @@ def _match_partner(
             partner_dict, chatter_msg, domain, partner_type_label)
         if partner:
             return partner
+        if partner_dict.get("gln"):
+            if "partner_identification_gln" in self.env.registry._init_modules:
```

Comment:
I would put the code in a separate method, to be easily extended in case the gln field is provided by a different module, other than `partner_identification_gln`. IMO most of the customers use the `gln` by just adding a field on partner.

So this block of code would be something like:

```python
        if partner_dict.get("gln"):
            partner = _match_partner_by_gln()
            if partner:
                return partner
```